releasewarrior/__main__.py

```diff
@@ -75,6 +75,8 @@ def parse_args():
                                help='taskcluster graphid used for release in question')
     parser_update.add_argument('--graphid-2', '--taskcluster-graphid-2', dest='graphid_2',
                                help='taskcluster graphid used for graph 2 of rc release in question')
+    parser_update.add_argument('--checkbox', choices=ALL_CHECKBOXES, dest='checkboxes',
```

Comment:
I figured I'd add the functionality first, and get rid of the other way afterwards if people were ok with it.  But I wanted a way to easily add new checkbox items, and this seemed to be easier than the other way.